### ISSUE TITLE
nethserver-freepbx: userman can't sync with AD users

### ISSUE THREAD

**Issue by Stell0** [NethServer/dev#5281]
**Steps to reproduce**
* configure NethServer to get users from AD (local or remote)
* Install nethserver-freepbx
* go to User Management FreePBX module 

**Expected behavior**

* List of users in AD shown. 

**Actual behavior**

No user shown
If syncing is forced manually with `/usr/bin/scl enable rh-php56 "/usr/sbin/fwconsole userman sync"` it returns 
```
[Adldap\Exceptions\Auth\BindException]  
Strong(er) authentication required 
```
If StartTLS is enabled manually, debugghing the ldap bind php command, we can also see that user used is user@localdomain instead of user@adrealm

**Components**

nethserver-freepbx
userman FreePBX module

**Comment by Stell0** [comment]
Now nethserver-freepbx correctly configures FreePBX Userman module to use TLS and right account suffix, btw we need to wait for FreePBX to accept this PR FreePBX/userman#6

Meanwhile, it's possible to work around this issue installing manually userman module from here https://github.com/Stell0/userman/tree/release/14.0 and launching /etc/e-smith/events/actions/nethserver-freepbx-conf-users

**Comment by gsanchietti** [comment]
Upstream has merged the PR.
Can we close this issue and release a new testing package?

**Comment by Stell0** [comment]
As stated by FreePBX, userman has been under heavy development, previous used AD and LDAP drivers became "Legacy" and new driver has been issued. This changed configuration method and options, making nethserver-freepbx configuration action obsolete. 
With last version of nethserver-freepbx, the action has been updated and now works with userman module >= 14.0.3.
**TL;DR** update nethserver-freepbx (from nethserver-testing nethserver-freepbx-0.0.1-1.116.gbf2e241.ns7.noarch.rpm), update userman from FreePBX module administration

**Note:** unauthenticated LDAP stil doesn't work
